tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py

```diff
@@ -18,15 +18,19 @@ class Marketplace(Base):
     _filter_locator = (By.ID, 'compat-filter')
     _marketplace_iframe_locator = (By.CSS_SELECTOR, 'iframe[src*="marketplace"]')
     _search_toggle_locator = (By.CSS_SELECTOR, '.header--search-toggle')
+
     name = 'Marketplace'
     manifest_url = 'https://marketplace.firefox.com/packaged.webapp'
 
-    def search(self, term):
+    def switch_to_marketplace(self):
```

Comment:
This method is currently not used outside, I'd mark it as protected.
Also, `switch_to_marketplace()` sounds either redundant (like `marketplace.switch_to_marketplace()`) or not detailed enough (what if a function in this page class, doesn't call that method?). Would `switch_to_app()` sound better?